types/node-netstat/index.d.ts

```diff
@@ -0,0 +1,118 @@
+// Type definitions for node-netstat 1.8
+// Project: https://github.com/danielkrainas/node-netstat#readme
+// Definitions by: Nick Glazer <https://github.com/nickglazer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export = node_netstat;
```

Comment:
```suggestion
export = node_netstat;
```
you may want to replace this with either `nodeNetstat` or just `netstat` and skip non JS snake case (dev ex)

Reply:
I can do that. I actually had it as `NodeNetstat` in the first commit of this PR. But, I wanted to try basing my `index.d.ts` more off of `dts-gen`, which output `node_netstat`. Thanks for the review.